android/SmartLock/app/src/main/res/layout/smart_lock_fragment.xml

```diff
@@ -2,14 +2,14 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
```

Comment:
This should probably be left as match_parent.  After version 8 fill_parent has be depreciated.   However, no need to change,  Since my code will overwrite it.